Makefile

```diff
@@ -53,7 +53,7 @@ test:
 #--------------------------
 .PHONY: newman-test
 newman-test:
-	newman run test/e2e-tests.json -e $(ENV_FILE) --verbose --delay-request 1000
+	newman run test/e2e-tests-staging.json -e $(ENV_FILE) --verbose --delay-request 10
```

Comment:
I'm not quite sure I understand. I think the end-to-end tests are used for development, pretest, and staging. Why do we need to have a staging suffix?

Reply:
My thinking was that we will want to add some smoke tests (e2e-tests-production) in the future so we can make sure some of the basic functionality is always working in prod.

Reply:
How about `e2e_tests_nonprod`?